api/src/handlers/users.ts

```diff
@@ -78,6 +91,13 @@ export const createUser = async (
   };
 
   try {
+    const uniqueStudentNumber = await UsersUtilities.uniqueStudentNumber(data.studentNumber);
+    if(!uniqueStudentNumber) {
```

Comment:
```suggestion
    if (!uniqueStudentNumber) {
```
some thing in other places